src/outfitMakerUse1.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+/**
+ * A sample use of the outfitMaker component (non-interactive)
+ */
+
+public class outfitMakerUse1 {
+
+    private String shirts;
+    private String pants;
+    private String shoes;
+    private ArrayList<OutfitMakerUse1> outfit;
+
+    public void OutfitMakerUse1(String shirt, String pants, String shoes) {
```

Comment:
I am somewhat confused by this because you don't import your component. You make a new one.